Podman cannot load stats from rootless containers

Reported: https://github.com/containers/libpod/issues/4268
Logs in console: `> warning: Failed to update container stats: {"error":"io.podman.ErrorOccurred","parameters":{"reason":"Link not found"}}`
Fails with: 
```
Traceback (most recent call last):
  File "test/check-application", line 441, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "test/check-application", line 477, in _testLifecycleOperations
    self.assertIn('%', cpu)
AssertionError: '%' not found in ''
```

verify/fedora-31
Ooops, it happened again<hr>
```
# testLifecycleOperationsUser (__main__.TestApplication)
#
Created symlink /home/admin/.config/systemd/user/sockets.target.wants/io.podman.socket -> /usr/lib/systemd/user/io.podman.socket.
Warning: Stopping cockpit.service, but it can still be activated by:
  cockpit.socket

DevTools listening on ws://127.0.0.1:9519/devtools/browser/5de626f8-aa3e-4cc0-b02f-871f2524f5e3
[1015/112020.458633:ERROR:egl_util.cc(60)] Failed to load GLES library: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: /usr/lib64/chromium-browser/swiftshader/libGLESv2.so: cannot open shared object file: No such file or directory
[1015/112020.460946:ERROR:viz_main_impl.cc(170)] Exiting GPU process due to errors during initialization
[1015/112020.464204:WARNING:gpu_process_host.cc(1188)] The GPU process has crashed 1 time(s)
> warning: grep: /sys/class/dmi/id/product_serial: Permission denied
grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error
grep: /sys/class/dmi/id/chassis_serial: Permission denied
grep: /sys/class/dmi/id/product_uuid: Permission denied

> warning: Failed to update container stats: {"error":"io.podman.ErrorOccurred","parameters":{"reason":"Link not found"}}
> warning: Unhandled event type  system
Wrote screenshot to TestApplication-testLifecycleOperationsUser-fedora-31-127.0.0.2-2501-FAIL.png
Wrote HTML dump to TestApplication-testLifecycleOperationsUser-fedora-31-127.0.0.2-2501-FAIL.html
Wrote JS log to TestApplication-testLifecycleOperationsUser-fedora-31-127.0.0.2-2501-FAIL.js.log
Journal extracted to TestApplication-testLifecycleOperationsUser-fedora-31-127.0.0.2-2501-FAIL.log
Connection to 127.0.0.2 closed by remote host.
Traceback (most recent call last):
  File "test/check-application", line 441, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "test/check-application", line 477, in _testLifecycleOperations
    self.assertIn('%', cpu)
AssertionError: '%' not found in ''

not ok 5 testLifecycleOperationsUser (__main__.TestApplication) # duration: 56s
```
First occurrence: 2019-10-15T14:05:50.867173
Times recorded: 1
Latest occurrences:

- 2019-10-15T14:05:50.867173

--- a/naughty/fedora-31/137-podman-stats-rootless
+++ b/naughty/fedora-31/137-podman-stats-rootless
@@ -1,4 +1,0 @@
-Failed to update container stats: {"error":"io.podman.ErrorOccurred","parameters":{"reason":"Link not found"}}
-*
-*b.wait(lambda: b.text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(5)") != "")
-

